### DISCUSSION TITLE
Stationary tests

### DISCUSSION THREAD

**Discussion by jbaron** [optimatika/ojAlgo#506] [General, ojAlgo and related code]
Would OjAlgo be a good foundation to build algorithms like ADF (Augmented Dickey Fuller) tests upon top?

**Comment by apete** [comment]
Don't know what that is, but googled it quickly.

I would assume there is some foundation to build on. Perhaps look at the ARCH/GARCH model stuff:

https://www.ojalgo.org/2022/07/generalized-autoregressive-conditional-heteroscedasticity/

**Comment by jbaron** [comment]
Thanks, that indeed looks like it could be a good starting point.
